### PR TITLE
Replace poetry with conda + pip

### PR DESCRIPTION
**Pull request by borisrakovan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,29 +19,31 @@ Our aim is to design a DL model that is pre-trained on visual imagery data (e.g.
...
-This project uses [poetry](https://python-poetry.org/) for dependency management. To install poetry, follow the instructions on the [poetry website](https://python-poetry.org/docs/#installation).
+This project uses [conda](https://docs.conda.io/en/latest/) and [pip](https://pip.pypa.io/en/stable/) for package management.
+
+```bash
 
 To create a virtual environment and install the dependencies, run
 
 ```bash
-poetry install --no-root
+$ conda create -n dtu_mlops_project python=3.11
```
Comment (miccio-dk) [inline README.md:29]: we could also store the environment using conda directly (in an `env.yaml` file extracted using `conda env export > env.yaml`. This way all these instructions could be simplified into `conda env create -f env.yaml`

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+torch >= 2.1.0
```
Comment (miccio-dk) [inline requirements.txt:1]: we maybe wanna use the official wheel for pytorch (and torchvision/audio) which is distributed on pytorch's official website.
there's some info at https://github.com/pytorch/pytorch/issues/29745#issuecomment-553588171 and/or https://stackoverflow.com/questions/60912744/install-pytorch-from-requirements-txt

File: README.md (hunk 1 above)
Comment (miccio-dk) [reply README.md:29]: forgot to mention: here's how to install pytorch + torchaudio/vision using conda (from https://pytorch.org/get-started/locally/): `conda install pytorch torchvision torchaudio pytorch-cuda=11.8 -c pytorch -c nvidia`

**Review by miccio-dk — COMMENTED** [review]
hey I added a few comments, let me know what do you think :)

File: README.md (hunk 1 above)
Comment (burndjs) [reply README.md:29]: I tried installing the branch br/conda with the new install instructions in the readme and that worked (i.e. make train worked). So I'd say this requirements.txt pip-based install is fine. In the past I had some experiences with a conda env.yaml where there were issues with transferring between different os.

Comment (miccio-dk) [reply README.md:29]: alright, let's stick to this then :)

File: requirements.txt (hunk 2 above)
Comment (miccio-dk) [reply requirements.txt:1]: also closing this

**Review by miccio-dk — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (borisrakovan) [reply README.md:29]: I also had an issue with conda when installing certain packages, they don't have some packages so I think pip's better

**Pull request merged**